lib/nerves_hub/devices.ex

```diff
@@ -545,11 +547,22 @@ defmodule NervesHub.Devices do
     end
   end
 
+  @spec ca_certificate_by_ski_match?(binary) :: boolean()
+  def ca_certificate_by_ski_match?(ski) do
```

Comment:
```suggestion
  @spec known_ca_ski?(binary) :: boolean()
  def known_ca_ski?(ski) do
```

Not set on this, but being more contextually based function name on what it is trying to accomplish vs _how_ it is accomplishing that might be a little more useful. What do you think?

Reply:
Fixed!